Kernel/Clause.cpp

```diff
@@ -383,7 +383,7 @@ std::string Clause::toString() const
   // print inference and ids of parent clauses
   result += " " + inferenceAsString();
 
-  if(env.options->proofExtra()!=Options::ProofExtra::OFF){
```

Comment:
This is a place I was using `-proof_extra free` for. To get more detailed clause output without having to pay for it.

Reply:
Fair enough - if you use it I'll put it back.